src/components/ChatDown/ChatDown.tsx

```diff
@@ -1,20 +1,27 @@
-// @ts-check
-import React, { useContext } from 'react';
-import PropTypes from 'prop-types';
+import React from 'react';
 
-import { LoadingChannels } from '../Loading';
-import { TranslationContext } from '../../context';
+import { LoadingChannels } from '../Loading/LoadingChannels';
 
 import placeholder from '../../assets/str-chat__connection-error.svg';
+import { useTranslationContext } from '../../context/TranslationContext';
+
+export type ChatDownProps = {
+  /** The type of error */
+  type: string;
```

Comment:
if we have a default value is it really required?

Reply:
nope, but again wasn't changing anything from what already was present